src/lib/extensions/extensionHost.ts

```
@@ -874,10 +939,34 @@ export class ExtensionHost {
+						} else if (!host.exportAudioCaptureLimitWarningLogged) {
+							host.exportAudioCaptureLimitWarningLogged = true;
+							console.warn(
+								`[ext:${extensionId}] Export sound capture limit (${EXPORT_CAPTURE_MAX_EVENTS}) reached; additional sound events are ignored`,
+							);
+						}
+					} else {
+						console.warn(
+							`[ext:${extensionId}] Skipping export sound capture: missing event time`,
+						);
+					}
+					return () => {};
```

Biome (2.4.15)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)